perceval/simulators/feed_forward_simulator.py

```diff
@@ -252,13 +260,13 @@ def _init_simulator(self, input_state: SVDistribution,
         # Now the Processor has only the heralds that were possibly added by adding Processors as input, all at the end
         heralded_dist = proc.generate_noisy_heralds()
         if len(heralded_dist):
-            input_state *= heralded_dist
+            input_state = input_state * heralded_dist
```

Comment:
It was because if you do this on a list, you modify the original function argument.
I just tested on a BasicState and it doesn't so I will revert this change